WrightTools/artists/_base.py

```diff
@@ -461,6 +462,73 @@ def imshow(self, *args, **kwargs):
             super().invert_yaxis()
         return out
 
+    def scatter(self, *args, **kwargs):
+        """Scatter plot a channel against two _variables_.
+        Scatter point color reflects channel values.
+        Data need not be structured.
+        If data object is not provided, scatter reverts to the [matplotlib parent method](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html?highlight=scatter#matplotlib.axes.Axes.scatter)
+
+        Parameters
+        ----------
+        data : 2D WrightTools.data.Data object
+            Data to plot.
+        x : int or string (optional)
```

Comment:
Remaining should be documented as key word only  (with some iffiness about x and y, as those _can_ be passed as positional, but according to current implementation  only in the "pass through" sense where we are not doing data things)

Reply:
if you want x and y to be passable as positional args, they need to handled as such (it appears you do based on your tests)